.github/workflows/update_protobufs.yml

```
@@ -6,6 +6,7 @@ permissions:
 
 env:
   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: "true"
+  NANOPB_VERSION: "0.4.9.1"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)